SQF/dayz_code/stringtable.xml

```diff
@@ -5462,7 +5462,7 @@
 		</Key>
 		<Key ID="STR_ITEMSODA_FULL_CLEAN_CODE_NAME_11">
 			<English>Beer</English>
-			<German>Dose (Bier)</German>
+			<German>Bier</German>
```

Comment:
Actually all the other translations are without the Can too but if you prefer the "Dose"-part then I'll add it back.